modules/azure-gaming/connected-resources/modules/ContainerService/managedClusters.bicep

```diff
@@ -0,0 +1,104 @@
+@description('Deployment Location')
+param location string
+param name string = 'k8-cluster'
+param agentPoolCount int = 3
+param agentPoolName string = 'k8agent'
+param vmSize string = 'Standard_L16s_v2'
+param nodeLabels string = 'horde-storage'
```

Comment:
Is this default label workload specific?

Reply:
```suggestion
param nodeLabels string = 'aks-nodes'
```

Reply:
Yes, changing the default, but will verify a default even makes sense...